### ISSUE TITLE
"Organize & Rename" function ignores {MediaInfo..} tags

### ISSUE THREAD

**Issue by desimaniac** [Sonarr/Sonarr#2036]
MediaInfo Tags (eg MediaInfo.Simple, MediaInfo.VideoCodec} are being ignored during manual renames, resulting in the codec (e.g. x264) being missing from the filenames.

![image](https://user-images.githubusercontent.com/5501908/27992490-5ecef69c-645b-11e7-8ce4-6cf46dab6541.png)

and

![image](https://user-images.githubusercontent.com/5501908/27992495-a5ced1ac-645b-11e7-924e-aae078ffe95f.png)

Sonarr Ver. 2.0.0.4856 dev

**Comment by markus101** [comment]
Are these files Sonarr imported or were already on disk when you started using Sonarr?
Do you have Analyze Video Files enabled (by default it is), but disabling it would stop Sonarr from getting media info from existing files andead to this exact behaviour.

**Comment by desimaniac** [comment]
These were the episodes Sonarr found and downloaded and then imported, and it did have the codec in it the filename. But soon after, I made a tweak to the filename scheme (ie added Episode Title tag), and when I went to manually rename via the "Organize & Rename" icon and saw that. 

I do have analyze video off, since I have this stuff stored on the cloud., but it did put the codec in during the first import. I thought that it looked at the filename.

**Comment by markus101** [comment]
Check the history for one of the episodes, that will indicate if Sonarr lost track of the file and then had to rescan it to import it again, which would mean a subsequent rename wouldn't have the media info because Analyze Video Files is disabled. I just verified that the media info tokens are working when media info is present. Currently the UI doesn't show any information that media info is present, but that will be possible in the v3 UI.

Closing this as there is no issue with the renaming tokens, if you want to troubleshoot how Sonarr lost track of the files take a look at the history and if you have debug logs, those as well, feel free to open a thread on the forums to troubleshoot further.
